src/widget/wmainmenubar.h

```diff
@@ -78,13 +81,15 @@ class WMainMenuBar : public QMenuBar {
     void slotDeveloperStatsBase(bool enable);
     void slotDeveloperDebugger(bool toggle);
     void slotVisitUrl(const QString& url);
+    void togglePracticemode(bool toggle);
 
   private:
     void initialize();
     void createVisibilityControl(QAction* pAction, const ConfigKey& key);
 
     UserSettingsPointer m_pConfig;
     ConfigObject<ConfigValueKbd>* m_pKbdConfig;
+    std::unique_ptr<ControlObject> m_pConfigPracticemodeEnabled;
```

Comment:
just make this a member:
`ControlObject m_pConfigPracticemodeEnabled;`

no need for it to be a pointer

Reply:
this comment is also wrong, sorry.  We usually make controlobjects pointers too (and then have to delete them).  Again, not sure why we haven't started migrating toward unique/shared pointers but it hasn't been the style yet.  Do we want to mix things?